front-end/src/App.js

```diff
@@ -1,21 +1,54 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import 'tachyons';
 
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
+	constructor() {
+		super()
+		this.state = {
+			songUrl: ''
+		}
+	}
+
+	performSearch = () => {
```

Comment:
Codeblocks can be skipped for a single line of code, just makes the code more readable IMO,